src/easeljs/geom/Point.js

```diff
@@ -99,6 +99,53 @@ this.createjs = this.createjs||{};
 		return this;
 	};
 	
+	/** 
+	 * Offsets the Point object by the specified amount. The value of dx is added to the original value of x to create the new x value. The value of dy is added to the original value of y to create the new y value.
+	 * @method offset
+	 * @param {Number} The amount by which to offset the horizontal coordinate, x.
```

Comment:
All the parameters are missing names. If you compile the docs, the first word after the type will be the parameter name. The correct syntax is:
`@param {type} name Description`
OR
`@param name {type} Description`.

Note you can also specify optional params, and default values:
`@param [type] name Optional Parameter`
`@param [type=value] name Optional parameter with default value]`

http://yui.github.io/yuidoc/syntax/